addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2786,7 +2786,7 @@ options.registry.ScrollButton = options.Class.extend({
      */
     updateUIVisibility: async function () {
         await this._super(...arguments);
-        if (this.$button.length && this.el.offsetParent === null) {
+        if (this.$button.length && this.findWidget('scroll_button_opt').$el.hasClass('d-none')) {
```

Comment:
Ugly before and ugly now ^^ `updateUIVisibility` is not supposed to update the DOM although I have a souvenir that I am the one who accepted this at the time I think. Could this maybe be improved to say "when we go back to a smaller height, disable the scroll button ?" explictely (via the height option)?

2-3 ways to do it I think (even though that's a more and more common think we have to improve), from most probable to be working to least probable:
- in the height option method, search for the scroll button option (`findWidget` / `requestUserValueWidgets`) and "click on it" (via the `enable`(?) method)
- in the height option method, notify the scroll option via an `option_update` trigger_up
- add a `data-trigger` on the height option button definition (probably not stable anyway)

Reply:
Well, actually this whole method can just be removed since https://github.com/odoo/odoo/commit/85ae7fe56e93fcc18b3842e6c116bcb6968988e3 which is already in charge of showing/hiding/detaching the button when you click on the height options.
I just removed that method.

> Ugly before and ugly now

That's a nice way to say it ;-). I did put it another way when I saw that line, probably because I didn't read the method comment at first, took me some time to figure what was that line actually doing.

Reply:
> can just be removed since https://github.com/odoo/odoo/commit/85ae7fe56e93fcc18b3842e6c116bcb6968988e3

Oops :grimacing: :+1: